docs/HowTo/Extending-EthSigner/Add-Signer.md

```diff
@@ -4,26 +4,26 @@ description: Adding an external signer to EthSigner
 
 # Creating an external signer for EthSigner
 
-EthSigner supports the implementation of additional external signers in same way as Hashicorp Vault.
+EthSigner supports the implementation of additional external signers in the same way as Hashicorp Vault.
```

Comment:
The Ethsigner source code is structured to allow additional signing mechanisms to be added by following an established pattern:

Reply:
By pattern do you mean the procedure that follows the statement to be changed?

Reply:
Fixed